Secrets/Sources/Domain/Services/BundleSecretsService.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  BundleSecretsService.swift
+//  
+//
+//  Created by Martin Dutra on 6/12/21.
+//
+
+import Foundation
+
+protocol BundleSecretsService {
```

Comment:
One concern I have is the amount of boilerplate code in the Secrets package. The core functionality is contained in `PlistService`, and consists of about 10 lines of code. However, the `cloc` tool is counting 150 lines of Swift code and 87 lines of comments in the Secrets pacakge:
```
github.com/AlDanial/cloc v 1.92  T=0.01 s (1344.1 files/s, 33602.1 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
Swift                           12             66             87            150
Markdown                         1             11              0             26
XML                              1              0              0             10
-------------------------------------------------------------------------------
SUM:                            14             77             87            186
-------------------------------------------------------------------------------
```

I am worried how this will scale when we apply it to bigger packages. One change that might help here would be the removal of `BundleSecretsService` and `SecretsServiceAdapter`. It looks like the only hard functionality they contribute is two lines of logging. If we moved these into `PlistService` it seems like it could adhere to `SecretsService` directly. I could see how having a `BundleSecretsService` layer could be useful if we supported several types of config files that we load from Bundles, but since we only support one it seems like we aren't letting our real interactions with the Secrets package drive the design.

How would you feel about slimming this down a little? Do you think we would still fit IDD principles in doing so?